<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestWoundDialog</class>
 <widget class="QDialog" name="RestWoundDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RestWoundDialog</string>
  </property>
  <widget class="QLabel" name="myCharacter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources/mainwindow.qrc">:/images/characterprofile/black_knight.jpg</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="counterGroupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>280</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : white</string>
   </property>
   <property name="title">
    <string>Select a Counter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="counterList">
      <property name="font">
       <font>
        <family>MS Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Heal</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/mainwindow.qrc"/>
  <include location="../Resources/mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
